hnn_core/gui/gui.py

```diff
@@ -549,7 +549,7 @@ class HNNGUI:
         in the network.
     """
 
-    def __init__(self, theme_color="#8A2BE2", log_window_height="100px",
+    def __init__(self, theme_color="#8A2BE2", log_window_height="200px",
```

Comment:
I'm sorry that's not what I meant ... I wanted to have the right side only for visualization and no logging window. So decrease the width of the logging window and increase the height of the visualization window

Reply:
Ah I see... will change this later tonight